### ISSUE TITLE
How to identify the first and last point in the swath path

### ISSUE THREAD

**Issue by Botronics** [Fields2Cover/Fields2Cover#60]
Hi @Gonzalo-Mier ,
Inside the swath-generated path, starting point and end point looks similar. How to identify the starting point and the endpoint.

**Comment by Gonzalo-Mier** [comment]
Not sure what you mean. Could you rephrase it a bit or add an image? Thanks

**Comment by Botronics** [comment]
hai @Gonzalo-Mier ,
In the fields2cover generated path, I tried to visualize it in google earth. The starting and end points look similar for the generated path. The paths are not connected. 
![new](https://user-images.githubusercontent.com/110804545/222070524-0216500c-0c8d-416f-ab55-f8032853833c.png)

**Comment by Gonzalo-Mier** [comment]
Should I close this issue?

**Comment by Botronics** [comment]
> Should I close this issue?

yes
